edgelet/hsm-sys/azure-iot-hsm-c/CMakeLists.txt

```diff
@@ -74,8 +74,11 @@ else()
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -std=c99 -Werror ")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fPIC -Werror ")
     else()
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -Wall -Werror -Wextra -Wformat=2 -Wformat-security -std=c99 ")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fPIC -Wall -Werror -Wextra -Wformat=2 -Wformat-security ")
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -std=c99 -Werror ")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fPIC -Werror ")
+
+        # set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fPIC -Wall -Werror -Wextra -Wformat=2 -Wformat-security -std=c99 ")
```

Comment:
Should we add a comment here to indicate why we are making this change?

Reply:
Yes. I will do that.